tools/rostest/src/rostest/rostest_main.py

```diff
@@ -98,6 +98,10 @@ def rostestmain():
     parser.add_option("--results-base-dir", metavar="RESULTS_BASE_DIR",
                       help="The base directory of the test results. The test result file is " +
                            "created in a subfolder name PKG_DIR.")
+    parser.add_option("--reuse-master", action="store_true",
```

Comment:
Most other options don't have a short option either. And `--reuse-master` is pretty uncommon. Since the original proposed `-l` does not make sense anymore I removed it. If you want to propose one I can easily update the PR and add a short option.